docker/dockerfiles/Dockerfile

```diff
@@ -56,12 +56,8 @@ USER ${USER}
 
 RUN mkdir -p "$(rbenv root)"/plugins && \
     git clone https://github.com/rbenv/ruby-build.git "$(rbenv root)"/plugins/ruby-build && \
-    rbenv install 3.0.5 && \
-    eval "$(rbenv init -)" && \
-    rbenv shell 3.0.5 && \
-    rbenv rehash && \
-    gem install bundler -v 2.3.22
```

Comment:
We're using the default bundler version rbenv gives for ruby 3.3.4.Even if you get the version wrong by a bit, `bundle install` will switch to whatever version is BUNDLED WITH in Gemfile.lock. Is there a reason this isn't safe? Could well be, but I didn't want to perpetuate complexity without understanding it.